metrics/report/report_dockerfile/tidy_scaling_nc.R

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env Rscript
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Show pod communication latency
+
+suppressMessages(suppressWarnings(library(ggplot2)))	# ability to plot nicely.
+suppressWarnings(suppressWarnings(library(ggpubr)))	# ggtexttable
+suppressMessages(library(jsonlite))			# to load the data.
+suppressMessages(library(scales))			# For de-science notation of axis
+library(tibble)						# tibbles for tidy data
+
+testnames=c(
+	"k8s-rapid-nc"
+)
+
+### For developers: uncomment following variables to run this as is in R
+# resultdirs=c("PATH/TO/RES1/", ...) # keep the ending slash on result paths
+# inputdir=""
```

Comment:
Do we need this comment? We don't have it in any of the other R files, which do the same thing. Generally I think this is covered by the use of the [report debug method](https://github.com/clearlinux/cloud-native-setup/tree/master/metrics/report#debugging-and-development), and the source'ing of the `Env.R` file..